Sources/CADCacheAdvance/CADCacheAdvance.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  Created by Dan Federman on 4/24/20.
+//  Copyright © 2020 Dan Federman.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+#if SWIFT_PACKAGE_MANAGER
+import CacheAdvance
+#endif
+import Foundation
+
+/// A cache that enables the performant persistence of individual messages to disk.
```

Comment:
It's a bummer we will need to keep comments like this in sync but it probably is good to add them here too.

Reply:
My thinking as well